### ISSUE TITLE
Project Info: Project Issue as Additional Tab

### ISSUE THREAD

**Issue by marcalwestf** [adempiere/adempiere#2211]
There is a new window Project Info, which displays important documents related:
![20181210_project_16](https://user-images.githubusercontent.com/1789408/49872649-53302a80-fddf-11e8-94b2-133659a192f7.png)

An additional tab for Project Issues which are related to the Project is missing.

**Comment by e-Evolution** [comment]
@marcalwestf , Yes is good idea adding the project issues , but using manufacturing order link to project , I think that the functionality for project issues not es useful, currently this parcial an incomplete , because exists a good integration with costing

**Comment by marcalwestf** [comment]
hi @e-Evolution Project Issue is part of Project Management.

Users can choose to implement it with Production, but they also are free to use Project Issue -for whatever reasons- and we should also cover such situation.

**Comment by e-Evolution** [comment]
@marcalwestf ,@marcalwestf , if in fact the reason for keeping it would be for backward compatibility, linking the inventory documents to a project is very impractical, what happen if lost some or never was adding, this functionality comes from the time of Compiere and when we did not have functionality for the management of production in a better way as manufacturing or production  light.

So I wonder if we should continue with the course of the project issues when the implementation of compiere seems to me that it was not completed and advanced as manufacturing or production  light..

Thus it may be more useful to work on production light  or manufacturing to support the work based on orders that are required for a project.

In the end, what the accounting is to solve is the charge to the project asset vs working progress  account for projects and consume of inventories.
